internals/core_models.py

```diff
@@ -287,3 +288,4 @@ class Stage(ndb.Model):
   enterprise_policies = ndb.StringProperty(repeated=True)
 
   archived = ndb.BooleanProperty(default=False)
+  created = ndb.DateTimeProperty(default=datetime.min, required=True)
```

Comment:
You could use `created = ndb.DateTimeProperty(auto_now_add=True)` so that it is automatically set.

Reply:
For some reason, doing that updates the created time of existing stages and messes up the ordering more.

Reply:
I updated the change to fix this